react/admin/ReturnDetails/components/ContactDetails.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+
+import { useReturnDetails } from '../../hooks/useReturnDetails'
+
+export const ContactDetails = () => {
+  const { data } = useReturnDetails()
+
+  if (!data) return null
+
+  const {
+    returnRequestDetails: {
+      customerProfileData: { name, email, phoneNumber },
+    },
+  } = data
+
+  return (
+    <section className="flex-ns flex-wrap flex-auto flex-column pt4 pb4">
+      <h3>
+        <FormattedMessage id="admin/return-app.return-request-details.contact-details.title" />
+      </h3>
+      <div className="mb5">
+        <p className="ma0">
+          <FormattedMessage id="returns.name" />: {name}
```

Comment:
Just adding here so we don't forget. Please change the message id to match the pattern we are following in the new version.